include/session/config/namespaces.hpp

```diff
@@ -39,6 +39,8 @@ namespace {
 
             case Namespace::RevokedRetrievableGroupMessages:
                 return "RevokedRetrievableGroupMessages";
+
+            default: return "Invalid";
```

Comment:
Style thing: it would be nicer to put this `return` after the end of the switch instead of in a `default` case, because that way this whole switch would start producing warnings for unhandled cases if new enum values get added that aren't handled here, which is a nice reminder.  (With the `default` that won't happen).

Reply:
Updated